nilearn/plotting/data/js/surface-plot-utils.js

```diff
@@ -76,22 +76,22 @@ function getCamera(plotDivId, viewSelectId) {
         }
     }
     let cameras = {
-        "left": {eye: {x: -1.7, y: 0, z: 0},
+        "left": {eye: {x: -3.0, y: 0, z: 0},
```

Comment:
the problem is that now the brain is too small and too far from the colorbar when plotting a surface map on a mesh in mni space (or close to it), which is the vast majority of use cases:

before:
![newplot](https://user-images.githubusercontent.com/9196501/129727928-010b75b8-bd32-4daa-9db9-bc1a7528d422.png)

now:

![newplot(1)](https://user-images.githubusercontent.com/9196501/129727942-402d9e0e-c02e-4e3c-8700-1a8c1691a98e.png)

I don't think deteriorating the plots for the most common use case to address an edge case is the way to go. maybe we should allow users to control the zoom (although they can do it with the mouse once the interactive plot is opened, and maybe further programmatic customization of the plots should be deferred to after #2902 is merged)

otherwise the fix to control the aspect ratio looks good!